internal/sops/keyservice/server.go

```diff
@@ -109,6 +256,18 @@ func (ks Server) Encrypt(ctx context.Context,
 		response = &keyservice.EncryptResponse{
 			Ciphertext: ciphertext,
 		}
+	case *keyservice.Key_AzureKeyvaultKey:
+		if _, err := os.Stat(filepath.Join(ks.HomeDir, "azure_kv.json")); os.IsNotExist(err) {
```

Comment:
`azure_kv.json` should be a `const` to ensure consistent usage.

Reply:
yes, you're right